indico/MaKaC/conference.py

```diff
@@ -15,10 +15,10 @@
 # along with Indico; if not, see <http://www.gnu.org/licenses/>.
 
 from itertools import ifilter
-from flask_pluginengine import plugin_context
 from sqlalchemy.orm import lazyload, joinedload, noload
 from werkzeug.urls import url_parse
 
+from indico.modules.events.cloning import LegacyEventCloner as EventCloner  # XXX update code importing it from here
```

Comment:
temporary, will go away once i updated the remaining cloners

http://programmers.stackexchange.com/questions/65467/what-does-xxx-mean-in-a-comment